core/tests/protractor_utils/CommunityDashboardTranslateTextTab.js

```diff
@@ -23,11 +23,15 @@ var CommunityDashboardTranslateTextTab = function() {
   var selectableLanguageElements = element(
     by.css('.protractor-test-language-selector'));
   var selectedLanguageElement = selectableLanguageElements.element(
-    by.css('option:checked'));
+    by.css('.oppia-translation-language-selector-inner-container'));
 
   var _selectLanguage = async function(language) {
+    await selectableLanguageElements.click();
```

Comment:
Add `await for element to be clickable` before this.

Reply:
Done